### PR TITLE
xxd: add page

### PR DESCRIPTION
**Pull request by agnivade**

### CODE REVIEW

**Comment by agnivade** [comment]
New page after a long time ! I may have gotten a bit rusty :stuck_out_tongue_closed_eyes: Please let me know how does it look. :smile:

File: pages/common/xxd.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# xxd
```
Comment (sbrl) [inline pages/common/xxd.md]: _That's_ what it's called! I've been looking for this command. I think I'm going to have to alias it...!

File: pages/common/xxd.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+> Create a hexdump from a binary file or vice-versa.
+
+- Generate a hexdump from a binary file and display the output to console:
+
+`xxd {{input_file}}`
```
Comment (sbrl) [inline pages/common/xxd.md]: I believe you can pipe to this command too. Not that it needs adding here - just a passing observation.

File: pages/common/xxd.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+`xxd -c {{8}} {{input_file}}`
+
+- Display in plain hex format without any gaps:
+
+`xxd -p {{input_file}}`
+
+- Revert a plaintext hex into binary and store it in an output file:
+
+`xxd -r -p {{input_file}} {{output_file}}`
```
Comment (sbrl) [inline pages/common/xxd.md]: Oooh coool! That's awesome. This makes xxd rather useful actually. You can go both backwards and forwards, and output in raw hex too! Very cool indeed.

Comment (agnivade) [inline pages/common/xxd.md]: Yep. :smile:

File: pages/common/xxd.md (hunk 2 above)
Comment (agnivade) [inline pages/common/xxd.md]: Yes you can. Went with the most common use case, which is having a "file".

**Pull request merged**